docs/local-development.md

```diff
@@ -128,6 +128,28 @@ In the example above, a `public/assets/style.css` file will be accessible at `ht
 
 > Be aware that serving assets in production will not work like this out of the box. You will need [to use a S3 bucket](/docs/runtimes/http.md#assets).
 
+### Blackfire
+
+The development FPM container comes with the blackfire extension. When using docker compose you can add following service configuration for the blackfire agent:
+
+```yaml
+services:
+  blackfire:
+    image: blackfire/blackfire
+    environment:
+      BLACKFIRE_SERVER_ID: server-id
+      BLACKFIRE_SERVER_TOKEN: server-token
+```
+
+Now you can enable the blackfire probe extension in your projects php.ini
+
+```ini
+extension=blackfire
+blackfire.agent_socket=tcp://blackfire:8707
+```
```

Comment:
If I understand correctly this must go in `php/conf.d/php.ini` right?

So that means this will break when deployed in production?

Reply:
Yes it would go. But with the PHP_INI_SCAN_DIR environment variable this would not be needed in the php.ini directly

Reply:
With latest changes it's possible to use a different ini file for development extensions